build/optaplanner-bom/pom.xml

```
@@ -7,6 +7,7 @@
     <groupId>org.optaplanner</groupId>
     <artifactId>optaplanner-parent</artifactId>
     <version>8.23.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
```

The `optaplanner-parent` is one level up:
```suggestion
    <relativePath>../../pom.xml</relativePath>
```